Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -28,15 +28,19 @@ CheckSource()
 	LogMsg "Running CheckSource"
 	current_clocksource="/sys/devices/system/clocksource/clocksource0/current_clocksource"
 
-	# Microsoft LIS installed version has lis_hyperv_clocksource_tsc_page or
-	#	lis_hv_clocksource_tsc_page.
-	# Without Microsoft LIS, hyperv_clocksource_tsc_page or hv_clocksource_tsc_page.
-	# CentOS 7.0 or older versions has hyperv_clocksource.
-	clocksource="v_clocksource_tsc_page"
-	source /etc/os-release
-	if [[ $DISTRO_NAME == "centos" || $DISTRO_NAME == "rhel" ]] && (($(echo "$VERSION_ID < 7.1" | bc -l))) || [[ $DISTRO == "redhat_6" || $DISTRO == "centos_6" ]]; then
-		clocksource="hyperv_clocksource"
+	# Microsoft LIS installed version for RH7.X has lis_hv_clocksource_tsc_page
+	# Without Microsoft LIS, hyperv_clocksource_tsc_page
+	clocksource="hyperv_clocksource_tsc_page"
+	mj=$(echo "$DISTRO_VERSION" | cut -d '.' -f 1)
+	mn=$(echo "$DISTRO_VERSION" | cut -d '.' -f 2)
+
+	if [[ $DISTRO_NAME == "centos" || $DISTRO_NAME == "rhel" || $DISTRO_NAME == "oracle" ]];then
```

Comment:
I don't think this change is correct for all distro and its versions.

Reply:
Here the distro check is only for RPM based distro for which LIS is supported. This check is to identify whether LIS is installed or not.

Reply:
I don't want to check-in the working code for the small range. If not, we need to spend hours of RCA of each failed case. We need to check all Azure images. Let me see.

Reply:
Randomly selected 27 images. 18 passed, 9 failed. Can we find why 9 distros failed?

Reply:
Internally PR test re-do and found 9 failure out of 27 random distros. @johnsongeorge-w  can fix the last issue next mont. :)

Reply:
Shared the updated result. We have only aborted for Ubuntu.